api/ccapi/include/model.h

```diff
@@ -192,6 +195,22 @@ class Model {
     const std::vector<std::string> &end_layers, ReferenceLayersType type,
     const std::vector<std::string> &type_properties = {}) = 0;
 
+  /**
+   * @brief Visit each layer inside model
+   * @param fn function to be called for each layer
+   * - call param
+   *   @param layer layer the call back is visiting
+   *   @param rc run layer context clubbed with layer, null if not initialized
```

Comment:
update doxygen as per updated signature and behavior - reference for run layer context, and add user_data.

Reply:
done.